lib/formtools/plugins/document.js

```diff
@@ -57,6 +75,19 @@ module.exports = function formtoolsPlugin (schema, options) {
         });
     }
 
+    // check if we need to render the default if grid, form and overview options are provided in object DSL
+    if (typeof options.grid !== 'function') {
+        var grid = pluginHelpers.gridDefaults(opts.grid, labels);
```

Comment:
With this and the next two, the variables should be locally scoped above on L18.